hardware/system/upboard/gaia/udev/99-gaia.rules

```diff
@@ -1,7 +1,7 @@
-KERNELS=="1-7.2:1.0", SYMLINK+="ttyJ4"
-KERNELS=="1-7.3:1.0", SYMLINK+="ttyJ5"
+KERNELS=="1-7.2:1.0", SYMLINK+="ttyUSB0"
+KERNELS=="1-7.3:1.0", SYMLINK+="ttyUSB1"
 KERNEL=="ttyS1", SYMLINK+="ttyJ7"
 KERNEL=="video[0-9]*", ATTR{name}="USB 2.0 Camera", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="8830", SYMLINK+="video-scan"
 KERNEL=="video[0-9]*", ATTR{name}="5M Camera ", ATTRS{idVendor}=="090c", ATTRS{idProduct}=="f37d", SYMLINK+="video-front"
-SUBSYSTEM=="video4linux", KERNELS=="1-2", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-scan"
-SUBSYSTEM=="video4linux", KERNELS=="1-3", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-front"
+SUBSYSTEM=="video4linux", KERNELS=="1-5", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-scan"
+SUBSYSTEM=="video4linux", KERNELS=="1-6", ATTRS{idVendor}=="0ac8", ATTRS{idProduct}=="0345", SYMLINK+="video-front"
```

Comment:
Aren't these only for the new UP 4000? Machines using the old board should stay the same right?

Reply:
Yeah, the files ended up switched between regular upboard and upboard-4000 directories. Should be fixed